src/luasocket.c

```diff
@@ -30,6 +30,7 @@
 #include "inet.h"
 #include "tcp.h"
 #include "udp.h"
+#include "netlink.h"
```

Comment:
ifdef

Reply:
I assume you meant for me to include the ifdef inside netlink.h (which I did), however, since theres no change in the line you tagged, your comment won't disappear.